helix-view/src/view.rs

```diff
@@ -97,9 +97,9 @@ impl View {
         const OFFSET: usize = 7; // 1 diagnostic + 5 linenr + 1 gutter
         let last_col = self.first_col + (self.area.width as usize - OFFSET);
 
-        if line > last_line.saturating_sub(scrolloff) {
+        if line > last_line.saturating_sub(scrolloff + 1) {
```

Comment:
maybe extract `last_line.saturating_sub(scrolloff + 1)` into a var?

Reply:
Actually, looking at the code again, I realized I was doing this in the wrong place.  The only reason the `+ 1` is needed there is because `last_line` was being computed incorrectly.  `last_col` was, too.  Fixed.